HW_Task1.py

```diff
@@ -0,0 +1,23 @@
+# Домашняя задача 1. Напишите программу, 
+# удаляющую из текста все слова, 
+# в которых присутствуют все буквы "абв",
+# но не обязательно в таком порядке.
+# Напр., АВтоБус 
+
+import re
+
+str = 'автобус едет в страну абвгдейка а затем в страну бавария'
+word = re.split('\W+', str)
```

Comment:
А что делает эта регулярка? Почему нельзя вот так:
```suggestion
word = str.split()
```